docs/components/pagination/README.md

```diff
@@ -124,8 +124,6 @@
 <template slot="Overview">
 <cdr-doc-table-of-contents-shell>
 
-Live code examples for **CdrPagination** can be found on [CodeSandbox](https://codesandbox.io/s/4jnw2l193x).
```

Comment:
pagination now has auto-generated sandboxes. also this codesandbox is wildly out of date 🙃